pom.xml

```diff
@@ -200,225 +249,305 @@
+            <scope>compile</scope>
+            <optional>true</optional>
         </dependency>
         <dependency>
+            <!-- BlockLocker -->
             <groupId>nl.rutgerkok</groupId>
             <artifactId>blocklocker</artifactId>
             <version>1.10</version>
             <type>jar</type>
-            <scope>system</scope>
-            <systemPath>${basedir}/lib/BlockLocker.jar</systemPath>
+            <scope>compile</scope>
+            <optional>true</optional>
         </dependency>
         <dependency>
+            <!-- LWCX (Lightweight Chests Extended) -->
             <groupId>com.griefcraft.lwc</groupId>
             <artifactId>LWCX</artifactId>
-            <version>2.2.6</version>
+            <version>2.2.7</version>
             <type>jar</type>
-            <scope>system</scope>
-            <systemPath>${basedir}/lib/LWCX.jar</systemPath>
+            <scope>compile</scope>
+            <optional>true</optional>
         </dependency>
-        <!-- only need ProtocolLib to support LibsDisguises -->
         <dependency>
+            <!-- ProtocolLib (to support LibsDisguises) -->
             <groupId>com.comphenix.protocol</groupId>
             <artifactId>ProtocolLib</artifactId>
             <version>4.7.0-SNAPSHOT</version>
             <type>jar</type>
-            <scope>system</scope>
-            <systemPath>${basedir}/lib/ProtocolLib.jar</systemPath>
+            <scope>compile</scope>
+            <optional>true</optional>
         </dependency>
         <dependency>
+            <!-- LibsDisguises -->
             <groupId>LibsDisguises</groupId>
             <artifactId>LibsDisguises</artifactId>
-            <version>10.0.25</version>
+            <version>10.0.25-SNAPSHOT</version>
             <type>jar</type>
-            <scope>system</scope>
-            <systemPath>${basedir}/lib/LibsDisguises.jar</systemPath>
+            <scope>compile</scope>
+            <optional>true</optional>
         </dependency>
         <dependency>
-            <groupId>net.ess3</groupId>
+            <!-- EssentialsX -->
+            <groupId>net.essentialsx</groupId>
             <artifactId>EssentialsX</artifactId>
-            <version>2.19.0</version>
+            <version>2.19.0-SNAPSHOT</version>
             <type>jar</type>
-            <scope>system</scope>
-            <systemPath>${basedir}/lib/EssentialsX.jar</systemPath>
+            <scope>compile</scope>
+            <optional>true</optional>
         </dependency>
         <dependency>
+            <!-- PlaceholderAPI -->
             <groupId>me.clip</groupId>
             <artifactId>placeholderapi</artifactId>
             <version>2.10.9</version>
-            <scope>provided</scope>
+            <type>jar</type>
+            <scope>compile</scope>
+            <optional>true</optional>
         </dependency>
         <dependency>
+            <!-- RedProtect-Core -->
```

Comment:
Why do we now need two dependencies for RedProtect?

Reply:
Because that was what the instructions were on its GitHub page for adding it as a dependency.